react/features/recording/components/Recording/AbstractStartRecordingDialog.js

```diff
@@ -266,12 +297,16 @@ class AbstractStartRecordingDialog extends Component<Props, State> {
  * @returns {{
  *     _appKey: string,
  *     _conference: JitsiConference,
+ *     _fileRecordingsServiceEnabled: boolean,
+ *     _isDropboxEnabled: boolean,
+ *     _fileRecordingsServiceSharingEnabled: boolean,
```

Comment:
Maybe use `_isFileRecording...` for consistency with `_isDropboxEnabled` ? At any rate, alpha sort.

Reply:
I got this name from Damyan, this is the actual config name that he'll use in config.js

Reply:
ok